schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -610,6 +610,10 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2017-09-01/Microsoft.Network.json#/resourceDefinitions/dnsZones_SOA" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-09-01/Microsoft.Network.json#/resourceDefinitions/dnsZones_SRV" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-09-01/Microsoft.Network.json#/resourceDefinitions/dnsZones_TXT" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2016-03-01/Microsoft.Insights.json#/resourceDefinitions/alertrules" },
```

Comment:
>{ "$ref": "https://schema.management.azure.com/schemas/2016-03-01/Microsoft.Insights.json#/resourceDefinitions/alertrules" }, [](start = 18, length = 125)

do you need this?

Reply:
i did not add this line. It was there from earlier.
all i did was, add a comma at the end of the line.

Reply:
you added this line :). If you search it in the file, it now appears twice.

Reply:
You are right. Sorry for the confusion. Thanks